packages/shared/src/components/post/PostComments.tsx

```diff
@@ -51,14 +54,14 @@ export function PostComments({
   const container = useRef<HTMLDivElement>();
   const { tokenRefreshed } = useContext(AuthContext);
   const { requestMethod } = useRequestProtocol();
-  const queryKey = generateQueryKey(RequestKey.PostComments, null, id);
+  const queryKey = generateCommentsQueryKey({ postId: id, sortBy });
```

Comment:
Personally not sure if this abstraction adds much, I'd prefer just to use `generateQueryKey` here as it needs to be as close to the query to avoid issues later on. 
(that said, also not strongly against it)

Reply:
Wanted to do it this way so nothing would be missed like I did. Forgot to realize that we have other areas that uses the same key.